docs/bmm-index.md

```
@@ -0,0 +1,224 @@
+npm run docs                         # Generate API docs (Swagger UI)
+
+# Code Quality
+npm run lint                         # Lint all code
+npm run format                       # Format all code
+```
+
+---
+
+## Project Structure at a Glance
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)